### ISSUE TITLE
Android version planned ?

### ISSUE THREAD

**Issue by rgarrigue** [duplicati/duplicati#2967]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Description

 
I just wanted to know if an Android version is planned :-)

**Comment by Pectojin** [comment]
I haven't heard anything new recently, but Kenneth addressed this [back in 2015](https://groups.google.com/forum/#!topic/duplicati/3zlbDcwzNdM) saying there weren't any plans due to a lot of difficulties backing up on mobile OS'es.

**Comment by Hand-of-Cthulhu** [comment]
Is it possible then to make something on android that can do just the restore function of duplicati? Being able to get some files from the cloud backup on your phone would be pretty useful.

**Comment by Pectojin** [comment]
Should be possible, but I don't know if anyone with android experience are interested in the project.

**Comment by Jackman3005** [comment]
> I haven't heard anything new recently, but Kenneth addressed this [back in 2015](https://groups.google.com/forum/#!topic/duplicati/3zlbDcwzNdM) saying there weren't any plans due to a lot of difficulties backing up on mobile OS'es.

This link is no longer valid. Is there another way to view this thread? Would love to have my phone backed up!

**Comment by Jackman3005** [comment]
Thanks for taking the time to explain it again! It makes sense and I understand the stance now. But it is rather unfortunate 😞

**Comment by 0x391F** [comment]
> No user installable app can read all files from other apps (such as settings, etc). This means that any app developed would only have very limited access to the data to back backup, and most likely would not be able to restore it anyway. There is no way to elevate an app to the sequirty level required as that would break the operating system security.
> 
> Due to this security model, it is only the phone OS vendor that can make an app which reads files from all apps.
> 
> tl;dr: only Apple, Google, Samsung etc has the access required to make a phone backup app. The "duplicati phone" is not on the roadmap ;).

I think at least duplicati could backup user's files, especially photos and videos. Apps data maybe not very important, beause some apps' data are storage on cloud, and some apps has been integrate backup feature.
